### PR TITLE
PROGRAMMES-6141 Allow counting of Episodes

### PR DESCRIPTION
**Pull request by votemike**
Required as part of https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6141

### CODE REVIEW

**Comment by votemike** [comment]
Test me please

**Comment by votemike** [comment]
Retest his please

**Comment by votemike** [comment]
Retest this please

File: src/Service/ProgrammesAggregationService.php [hunk 1]
```diff
@@ -30,6 +30,14 @@ public function __construct(
         parent::__construct($repository, $mapper, $cache);
     }
 
+    public function countStreamableOnDemandEpisodes(
+        Programme $programme,
+        $ttl = CacheInterface::NORMAL,
+        $nullTtl = CacheInterface::SHORT
+    ): int {
+        return $this->countStreamableDescendantsByType($programme, 'Episode', $ttl, $nullTtl);
```
Comment (felipeparaujo) [inline src/Service/ProgrammesAggregationService.php]: Why the split here?

Comment (votemike) [reply src/Service/ProgrammesAggregationService.php]: To be consistent with `findStreamableDescendantsByType()`

**Pull request merged**